lib/swarm/tracker/tracker.ex

```diff
@@ -17,22 +17,39 @@ defmodule Swarm.Tracker do
   alias Swarm.Registry
   alias Swarm.Distribution.Strategy
 
+  defmodule Tracking do
+    @type t :: %__MODULE__{
+      name: term(),
+      m: atom(),
+      f: function(),
+      a: list(),
+      from: {pid, tag :: term},
+    }
+    defstruct name: nil,
```

Comment:
Since all of the default values are `nil`, let's make this `defstruct [:name, :m, :f, :a, :from]`